plugins/music.js

```diff
@@ -5,11 +5,23 @@ module.exports = function(pluto) {
 
     var musicModule = {
         lastPlaying: null,
+        progress: null,
         queue: [],
         paused: false,
         lastMessage: null
     };
 
+    pluto.addListener("player::progress", function(data) {
```

Comment:
Do we want to just replace all polling with websockets, or work more on making the polling a little less sketchy? If it's the former I'm inclined to leave this as is for now if it's going to get redone anyway